### ISSUE TITLE
FEAT: Add Server Restart reason

### ISSUE THREAD

**Issue by HilkopterBob** [HilkopterBob/PackageLock#58]
Add a message channel to push a restart reason into the server Go-Routine, so we can print it out to the user and or logs

We need to touch following code:
https://github.com/HilkopterBob/PackageLock/blob/e756cce7879ffb5dddd8317651dacd69f5cb5bd3/main.go#L26-L30

and:
https://github.com/HilkopterBob/PackageLock/blob/e756cce7879ffb5dddd8317651dacd69f5cb5bd3/main.go#L226-L247

Do you want to look into that @Kartoffel096 ?

**Comment by saxenaaman628** [comment]
@HilkopterBob, Hi can I contribute to this

**Comment by HilkopterBob** [comment]
Yes absolutely! But keep in mind this project is in its early stages. But other than that your work is greatly appreciated!

**Comment by saxenaaman628** [comment]
Hi, Is there any group channel, Where I can directly connect with you? and ask questions.
